packages/core/src/CorePlugin.ts

```diff
@@ -44,6 +47,9 @@ export class CorePlugin extends JWPlugin {
         applyFormat: {
             handler: this.applyFormat.bind(this),
         },
+        list: {
```

Comment:
I guess you already thought about using `createList` but chose to use `list` instead, I'm looking forward to knowing why :)